geniza/corpus/models.py

```diff
@@ -386,11 +386,24 @@ def iiif_urls(self):
             )
         )
 
+    def has_transcription(self):
+        """Admin display field indicating if document has a transcription."""
+        return any(note.has_transcription() for note in self.footnotes.all())
+    has_transcription.short_description = "Transcription"
+    has_transcription.boolean = True
+    has_transcription.admin_order_field = "footnotes__content"
+
+    def has_image(self):
+        """Admin display field indicating if document has a IIIF image."""
+        return any(self.iiif_urls())
+    has_image.short_description = "Image"
+    has_image.boolean = True
+    has_image.admin_order_field = "textblock__fragment__iiif_url"
+
     @property
     def title(self):
         """Short title for identifying the document, e.g. via search."""
-        # NOTE preliminary, pending more discussion
-        return f"{self.doctype or 'Unknown'} ({self.id})"
+        return f"{self.doctype or 'Unknown'}: {self.shelfmark_display or '??'}"
```

Comment:
Isn't this still preliminary? It doesn't look quite like the format we finally settled on. (Also out of scope!)

Maybe you can make an issue for revising document title for the public site and note there's a preliminary implementation